openedx/stanford/pavelib/i18n.py

```diff
@@ -144,23 +79,8 @@ def stanfordi18n_transifex_push():
     """
     Push source strings to Transifex for translation
     """
-    execute('tx push -s -r "stanford-openedx.*"')
-    print('Pushed source files to Transifex')
-
-    # merge in existing platform translations
-    for lang in CONFIG.translated_locales:
-        if not CONFIG.get_messages_dir(lang).exists():
-            # Language not yet available in code, fetch from upstream Transifex
-            print('Fetch upstream translations manually for ' + lang)
-            break
-
-        merge_mappings = {
-            'django.po': ['django', 'mako'],
-            'djangojs.po': ['djangojs', 'underscore'],
-        }
-        for existing, targets in merge_mappings.items():
-            for target in targets:
-                merge_existing_translations(existing, target, lang)
+    push()
+    merge_translations()
```

Comment:
The changed helper function name is not being imported, alternatively can just use my original function name and import `merge_existing_translations`